packages/office-ui-fabric-react/src/components/__snapshots__/DetailsList.CustomFooter.Example.tsx.shot

```diff
@@ -2,314 +2,14 @@
 
 exports[`Component Examples renders DetailsList.CustomFooter.Example.tsx correctly 1`] = `
 <div>
-  <div>
```

Comment:
is this huge change here intended?

Reply:
Yes, because I removed a bunch of excessive complexity (not related to custom footers, just copied from another example)